unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -930,7 +930,7 @@ notifyUser dir o = case o of
       CouldntLoadRootBranch repo hash ->
         P.wrap $
           "I couldn't load the designated root hash"
-            <> P.group ("(" <> fromString (Hash.showBase32Hex hash) <> ")")
+            <> P.group ("(" <> P.text (Hash.base32Hex $ Causal.unRawHash hash) <> ")")
```

Comment:
Hash.showBase32Hex was a weird function that was only called here, and I don't think did what we'd have expected it to do here anyway, so now we call something simpler.